tests/reference/llvm-print_04-443a8d8.stdout

```diff
@@ -1,10 +1,10 @@
 ; ModuleID = 'LFortran'
 source_filename = "LFortran"
 
-@u = global i64 -922337203685477580
-@x = global i32 -2147483648
-@y = global i16 -32768
-@z = global i8 -128
+@u = global i64 0
+@x = global i32 0
+@y = global i16 0
+@z = global i8 0
```

Comment:
I think I would keep these initializations here. This is relatively minor, so we can do it in the next PR.

Reply:
I created an issue for this at https://github.com/lcompilers/lpython/issues/1582.